SPECS/libarchive/libarchive.spec

```diff
@@ -60,6 +60,9 @@ make %{?_smp_mflags} check
 %{_libdir}/pkgconfig/*.pc
 
 %changelog
+* Tue Jun 25 2024 Neha Agarwal <nehaagarwal@microsoft.com> - 3.7.4-1
+- Update to v3.7.4 to fix CVE-2024-26256 and CVE-2024-37407
```

Comment:
The 3.0 dev branch has been recently locked down to include only changes blocking the 3.0 GA. When it comes to CVEs, it looks like we're also preferring patches to versions upgrades to keep things stable. Considering that, is it possible to use a patch instead?